src/ixdat/readers/zilien.py

```diff
@@ -186,15 +439,20 @@ def read(self, path_to_spectrum, cls=None, **kwargs):
     from pathlib import Path  # noqa
     from ixdat.measurements import Measurement  # noqa
 
-    path_to_test_file = Path.home() / (
-        "Dropbox/ixdat_resources/test_data/"
-        # "zilien_with_spectra/2021-02-01 14_50_40.tsv"
-        "zilien_with_ec/2021-02-01 17_44_12.tsv"
+    path_to_test_file = (
+        Path(__file__).parent.resolve().parent.parent.parent
+        / "test_data"
+        / "Zilien version 1"
+        / "2022-04-06 16_17_23 full set.tsv"
     )
 
-    ecms_measurement = Measurement.read(
+    ecms_measurement = ECMSMeasurement.read(
```

Comment:
why the change here?
`Measurement.read()` will be the default way to start in ixdat, and should do the right thing as much as possible!

Reply:
Agreed. I think this was just for testing. I will adjust it and put it in a function etc.